source/jormungandr/jormungandr/interfaces/v1/Journeys.py

```diff
@@ -156,7 +159,8 @@ def wrapper(*args, **kwargs):
                     args['direct_path'] = 'only' if 'non_pt' in journey['tags'] else 'none'
                     args['min_nb_journeys'] = 5
                     args['is_journey_schedules'] = True
-                    allowed_ids.update(args.get('allowed_id[]', []))
+                    param_values = allowed_id_args.get('allowed_id[]') or []
+                    allowed_ids.update(param_values)
```

Comment:
1. Is it possible to add a unit test on that ? :pray:

2. I suspect the final `[]` might disturb flask, and I think we should add tests for similar params on all endpoints (maybe only manual if automated tests are not possible).

Reply:
:+1:

Reply:
Since raw_args is a NameSpace but not a dict, 
- raw_args.get('allowed_id[]', []) gives **None** if parameter **allowed_id[]** is absent.
- raw_args.get('allowed_id[]') or [] gives **[]** if parameter **allowed_id[]** is absent.

Reply:
Thanks for the tests !